WcaOnRails/.rubocop_todo.yml

```diff
@@ -234,6 +234,7 @@ Style/ClassAndModuleChildren:
     - 'app/controllers/accounts/registrations_controller.rb'
     - 'app/controllers/api/v0/api_controller.rb'
     - 'app/controllers/api/v0/competitions_controller.rb'
+    - 'app/controllers/api/v0/records_controller.rb'
```

Comment:
I had a chat with pedro about this and he was just as confused.

I was seeing it as a way that we don't have to write module twice in the instead of API:V0:xyzController.

Two things we could do: change rubocop so that we can use the A::B::C notation or fix these files.